app/api/v1/models.py

```diff
@@ -0,0 +1,42 @@
+users = []
+political_parties = []
+political_offices = []
+
+class SaveUser():
```

Comment:
I think this classs can just be named User
```suggestion
class User():
```